bin/ruby-build

```diff
@@ -748,7 +748,18 @@ build_package_truffleruby() {
   build_package_copy
 
   cd "${PREFIX_PATH}"
-  "${PREFIX_PATH}/lib/truffle/post_install_hook.sh"
+  "${PREFIX_PATH}/lib/truffle/post_install_hook.sh" || return $?
+}
+
+build_package_graalvm() {
+  build_package_copy
+
+  cd "${PREFIX_PATH}"
+  bin/gu install ruby || return $?
+
+  local ruby_home
+	ruby_home=$(bin/ruby -e 'print RbConfig::CONFIG["prefix"]')
+	"$ruby_home/lib/truffle/post_install_hook.sh" || return $?
```

Comment:
What is the purpose for the || return $? ? It's the last line of the function so the last line's exit status will be the exit status for the function, no?

Reply:
You're right, it's probably unneeded if it stays the last line.